aspnetcore/grpc/health-checks.md

```diff
@@ -77,21 +77,34 @@ builder.Services.Configure<HealthCheckPublisherOptions>(options =>
 
 ## Call gRPC health checks service
 
-The [`Grpc.HealthCheck`](https://www.nuget.org/packages/Grpc.HealthCheck) package includes a client for gRPC health checks:
+The [`Grpc.HealthCheck`](https://www.nuget.org/packages/Grpc.HealthCheck) package includes a client for gRPC health checks. The client factory approach can be utilized to configure the client and make health check calls.
 
-```csharp
```

Comment:
I think it is best to leave the start as it is. It provides a simple, complete example of creating a client and using a client. At the end say that the health check client can be used with the client factory and show the new snippit of code.